src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -112,6 +112,12 @@ final class SSLConfiguration implements Cloneable {
     static final int maxCertificateChainLength = GetIntegerAction.privilegedGetProperty(
             "jdk.tls.maxCertificateChainLength", 10);
 
+    // Limit the maximum certificate chain length accepted from clients
```

Comment:
Should these be moved to after line 89?

Reply:
Remain to stay after the related maxCertificateChainLength.